src/slide/slide.config.ts

```diff
@@ -0,0 +1,44 @@
+import { ThyUpperOverlayConfig, ThyUpperOverlayOptions } from '../core/overlay';
+import { InjectionToken } from '@angular/core';
+
+export type ThySlideFromTypes = 'left' | 'right' | 'top' | 'bottom' | 'void' | 'exit';
+
+export class ThySlideConfig extends ThyUpperOverlayConfig {
+    key?: string; // 尽量不要使用key,使用id,将要废弃
+
+    from?: ThySlideFromTypes;
+
+    class?: string;
+
+    hasBackdrop?: boolean;
+
+    initialState?: Object;
```

Comment:
remove hasBackdrop and initialState, because there has exist in ThyUpperOverlayConfig